extensions/raydocs/src/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { List } from "@raycast/api";
+import { useState } from "react";
+import LinkItem from "./components/LinkItem";
+import { Link } from "./types";
+import { scrapDocs } from "./utils";
```

Comment:
Should `scrapDocs` be `scrapeDocs`?

Reply:
It could be getLinks or something simmilar 🤔

Reply:
Sounds good, getLinks, scrapeMenuLinks, scrapeLinks, fetchLinks, fetchMenuLinks, etc. could all be decent options 😁

Reply:
Done 🧑‍🚒